### ISSUE TITLE
why performance is different from what is reported?

### ISSUE THREAD

**Issue by isalirezag** [jwyang/faster-rcnn.pytorch#531]
I am using the pretrained weights that is provided on website `faster_rcnn_1_6_10021.pth` but i get different performance from what is reported on the website. am i doing something wrong?

**Comment by EMCP** [comment]
might help you post your performance , in order to debug

**Comment by isalirezag** [comment]
Sure, here is what i got:
```
Results:
0.656
0.775
0.610
0.460
0.454
0.769
0.769
0.839
0.470
0.678
0.606
0.767
0.811
0.692
0.730
0.371
0.606
0.651
0.741
0.709
0.658
~~~~~~~~
```
but they mentioned that `faster_rcnn_1_6_10021` should be around 70. Am i doing some mistake?

**Comment by AlexanderHustinx** [comment]
Are you using pytorch-1.0?

**Comment by isalirezag** [comment]
The overal precision is the last number which is 0.658. 
Yes I am using pytorch 1.0

**Comment by AlexanderHustinx** [comment]
I'll refer you to the observations of this issue:
https://github.com/jwyang/faster-rcnn.pytorch/issues/419

It seems that PyTorch-1.0 and PyTorch-0.4.0 have a slight mismatch in their way to training the model.
So the results are re-usable, but they will be off.

This leaves you with 3 options:
1. Revert to Pytorch-0.4.0 and use the trained model
2. Retrain the model yourself with the mentioned parameters (very similar accuracy)
3. Use the PyTorch-0.4.0 trained model for the PyTorch-1.0 code (worse accuracy, as you mentioned)

**Comment by kangkang59812** [comment]
almost same as you.Did you train the model youself? And did you get the mAp=70.6 or around it?
